src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -815,9 +814,9 @@ impl<'a> Resolver<'a> {
                             break;
                         }
                         MetaItemKind::List(nested_metas) => for nested_meta in nested_metas {
-                            match nested_meta.word() {
-                                Some(word) => single_imports.push((word.name(), word.span)),
-                                None => ill_formed(nested_meta.span),
+                            match nested_meta.ident() {
+                                Some(ident) if nested_meta.is_word() => single_imports.push(ident),
```

Comment:
For a follow up PR, I see this pattern in multiple places, it'd be nice to add a method `get_word_ident()` to `nested_meta` so you can use `if let` everywhere the alternative is discarded/an error.